### DISCUSSION TITLE
Meetup

### DISCUSSION THREAD

**Discussion by 0xfe** [0xfe/vexflow#881] [General]
There's a lot going on in the codebase right now, so we should have a friendly chat. Anyone around in the next hour or so?

@rvilarl @wassertim @mscuthbert ping

Update: Let's meet at https://meet.google.com/non-ewtv-spt at 5:30 Eastern Time (UTC−04:00)

**Comment by mscuthbert** [comment]
Sure thing!  :-)  Just reply here where we’ll meet.

On Apr 6, 2021, at 11:13, Mohit Cheppudira ***@***.******@***.***>> wrote:



There's a lot going on in the codebase right now, so we should have a friendly chat. Anyone around in the next hour or so?

@rvilarl<https://github.com/rvilarl> @wassertim<https://github.com/wassertim> @mscuthbert<https://github.com/mscuthbert> ping

**Reply by 0xfe** [reply]
Great! Let's meet at https://meet.google.com/non-ewtv-spt in about 15 mins.

**Reply by wassertim** [reply]
> Great! Let's meet at https://meet.google.com/non-ewtv-spt in about 15 mins.

That is really is a short notice. I will skip my bed time then :)

**Comment by 0xfe** [comment]
On Tue., Apr. 6, 2021, 17:25 Tim Tkachenko, ***@***.***>
wrote:

> Great! Let's meet at https://meet.google.com/non-ewtv-spt in about 15
> mins.
>
> That is really is a short notice. I will skip my bed time then :)
>

I know, apologies. A glass of scotch might help :-)

**Reply by rvilarl** [reply]
Hi it was a pleasure to get to know you yesterday.
Have a great day!
